lib/pages/Profile.dart

```diff
@@ -30,150 +25,146 @@ class _ProfileState extends State<Profile> {
-                            onPressed: () {
-                              Navigator.push(
-                                  context,
-                                  MaterialPageRoute(
-                                      builder: (context) => EditProfile(driverProvider.driver)
-                                  )
-                              );
-                            },
-                          )
-                        ]),
-                        Positioned(
-                          child: Text(
-                              "Joined 03/2020",
-                              style: TextStyle(
-                                fontSize: 14,
-                                color: Color.fromRGBO(132, 132, 132, 1),
-                              )
-                          ),
-                          top: 45,
+                            SizedBox(height: 4),
+                            Text('Joined ' + driverProvider.driver.startDate,
+                                style: TextStyle(
+                                  fontSize: 14,
+                                  color: Theme.of(context).accentColor,
+                                )
+                            )
+                          ],
                         )
-                      ],
-                    )
-                )
-              ]
-              )
-          ),
-          SizedBox(height: 6),
-          InfoGroup(
-            "Account Info",
-            [
-              InfoRow(
-                "email",
-                Icons.mail_outline,
-                driverProvider.driver.email,
+                      ]
+                  )
               ),
-              InfoRow("phone number", Icons.phone,
-                  driverProvider.driver.phoneNumber)
+              sectionDivider,
+              InfoGroup(
+                'Account Info',
+                [
+                  InfoRow(
+                    Icons.mail_outline,
+                    driverProvider.driver.email,
+                  ),
+                  InfoRow(
+                      Icons.phone,
+                      driverProvider.driver.phoneNumber,
+                      editPage: EditPhoneNumber(driverProvider.driver.phoneNumber)
+                  ),
+                  InfoRow(
+                    Icons.person,
+                    driverProvider.driver.firstName + ' ' + driverProvider.driver.lastName,
+                    editPage: EditName(driverProvider.driver.firstName, driverProvider.driver.lastName),
+                  ),
+                ],
+              ),
+              sectionDivider,
+              // commenting out right now because this is likely being removed or pushed to v2
+              //SettingRow('Notifications', 'Set your notification preferences', NotificationSettings()),
+              //TODO: implement this
+              SettingRow('Legal', 'Terms of Service & Privacy Policy', Container()),
```

Comment:
Do we know what we are going to include in the page this directs us to?

Reply:
This'll be a page with the links to https://www.dfa.cornell.edu/policy/policies/access-student-information and https://www.dfa.cornell.edu/policy/policies/data-stewardship-and-custodianship but we're still waiting on designs and Mihikaa has been assigned to this